packages/grafana-ui/src/components/VizLayout/VizLayout.tsx

```diff
@@ -45,6 +47,10 @@ export const VizLayout: VizLayoutComponentType = ({ width, height, legend, child
 
   const legendStyle: CSSProperties = {};
 
+  const legendSpacing: CSSProperties = {
+    marginLeft: theme.spacing(),
+  };
```

Comment:
do we need this margin? think we can just remove it from VizLegendTable

Reply:
I mean it's not needed when legend is at the bottom

Reply:
@torkelo i assumed it was there to give some space between the visualisation and the legend, but if we think there's enough space already then yeah we can just remove it.

Reply:
ah just saw the other comment about not being needed at the bottom... agreed. so can either remove it entirely or put some logic in to conditionally apply it when right aligned. we already have some styling logic based on the alignment so that is a small change tbh.